include/server_table.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * @file server_table.h
+ * @brief Provides hash table for server cert lookup.
+ */
+
+#ifndef INCLUDE_SERVER_TABLE_H_
+#define INCLUDE_SERVER_TABLE_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "charbuf.h"
+
+typedef struct ServerEntry
+{
+  charbuf server_id;
+  charbuf cert;
+} CertEntry;
+
+typedef struct Certs
+{
+  CertEntry *entries;
+  size_t num_entries;
+  size_t mem_size;
+} ServerTable;
+
+extern ServerTable server_table;
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/**
+ * <pre>
+ * This function to add values in hash table based on location in server_id.
+ * </pre>
+ *
+ * @param[in] server_id.chars Server identifier assumed to be null terminated
+ * @param[in] server_id.len The length of the server identifier
+ * @param[in] handle The handle value for the cert data location in the kmyth unseal data table
+ * @param[in] server_table The server table that the new cert needs to be added to
```

Comment:
This function doesn't appear to take the ```server_table``` as an argument.

Reply:
"'server_table'" is an argument for the function and is used in the function.  The function intakes "'server_table'" as an extern variable so you don't have to call the variable as an input argument which is the same way we do it for the key table.